src/components/Onboarding/steps/SelectPIN/index.js

```diff
@@ -25,20 +44,20 @@ export default (props: StepProps) => {
           category="Onboarding"
           name="Choose PIN"
           flowType={onboarding.flowType}
-          deviceType={onboarding.isLedgerNano ? 'Nano S' : 'Blue'}
+          deviceType={cleanDeviceName(onboarding.deviceType)}
```

Comment:
i'm a bit confused why we don't pass-in deviceType, which is original a model.id , now you give a `Nano X` string? 🤔